### PR TITLE
Updating privacy policy

### PR DESCRIPTION
**Pull request by mgamini**
tenor-22870003

Signed-off-by: Garrett Amini <kadjar@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by cnunciato** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/341732970).

File: www/config.rb [hunk 1]
```diff
@@ -224,3 +224,4 @@ def render_markdown(text)
 redirect 'demo/packaging-system/index.html', to: '/demo/packaging-system/steps/1'
 redirect 'demo/build-system/index.html', to: '/demo/build-system/steps/1'
 redirect 'demo/process-supervisor/index.html', to: '/demo/process-supervisor/steps/1'
+redirect 'legal', to: 'legal/licensing/'
```
Comment (cnunciato) [inline www/config.rb]: I think this has to be `legal/index.html`, but acceptance should tell us for sure. :smile:

Comment (cnunciato) [reply www/config.rb]: Yeah looks like that does need to be there:

http://www.acceptance.habitat.sh/legal

Changing `'legal'` to `'legal/index.html'` does the trick, verified locally.

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by mgamini** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/341996745).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/341996745).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/341996745).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/341996745).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/341996745) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/341997800).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/341997800).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/342000155).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/342000155).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/342000155).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/342000155).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**